local.sh

```diff
@@ -6,24 +6,23 @@
 export SOURCE_REPO_DIR=../aws-sdk-ios
 export XCF_OUTPUT_DIR=xcframeworks/output/XCF
 
+# clear the output
```

Comment:
Add this file to ignore list in `package.swift` ?

Reply:
This file is at the root and not in a target directory. It does not have to be excluded.